visualizations/frontend/filters/v2/TRangeInput.vue

```diff
@@ -76,14 +76,18 @@
         },
         methods: {
             update(e) {
-                if (e.minValue === e.min && e.maxValue == e.max) {
-                    this.unsetFilterValue('min', true);
-                    this.unsetFilterValue('max', true);
-                } else {
-                    this.barMinValue = e.minValue;
-                    this.barMaxValue = e.maxValue;
-                }
-            }
+                if (this.throttleInterval) clearInterval(this.throttleInterval);
+
+                this.throttleInterval = setTimeout(() => {
+                    if (e.minValue === e.min && e.maxValue == e.max) {
+                        this.unsetFilterValue('min', true);
+                        this.unsetFilterValue('max', true);
+                    } else {
+                        this.barMinValue = e.minValue;
+                        this.barMaxValue = e.maxValue;
+                    }      
+                }, 500)
+            },
```

Comment:
Nice improvement here! I just have a few comments:
- `throttleInterval` does not seem a good name for this variable. I would call it something like `throttleId` or similar.
- I think than 500ms is way too much, we should be good with 250ms.
- I would like to avoid one liners `if`. This is a 100% preference but I prefer to always write the code the same way no matter the amount of lines of its body:
```js
if (this.throttleInterval) {
    clearInterval(this.throttleInterval);
}
```

Reply:
Good points, thanks @egcastro, updating.